dojo/tools/openvas_csv/parser.py

```diff
@@ -77,92 +74,11 @@ def __init__(self):
-                url)
-
-            if rhost:
-                protocol = o.scheme
-                host = o.netloc
-                path = o.path
-                query = o.query
-                fragment = o.fragment
-
-                port = 80
-                if protocol == 'https':
-                    port = 443
-
-                if rhost[11] is not None:
-                    port = rhost[11]
-
-                try:
-                    dupe_endpoint = Endpoint.objects.get(protocol=protocol,
-                                                         host=host + (":" + port) if port is not None else "",
-                                                         query=query,
-                                                         fragment=fragment,
-                                                         path=path,
-                                                         product=finding.test.engagement.product)
-                except:
-                    dupe_endpoint = None
-
-                if not dupe_endpoint:
-                    endpoint = Endpoint(protocol=protocol,
-                                        host=host + (":" + str(port)) if port is not None else "",
-                                        query=query,
-                                        fragment=fragment,
-                                        path=path,
-                                        product=finding.test.engagement.product)
-                else:
-                    endpoint = dupe_endpoint
-
-                if not dupe_endpoint:
-                    endpoints = [endpoint]
-                else:
-                    endpoints = [endpoint, dupe_endpoint]
-
-                finding.unsaved_endpoints = endpoints
-
-        # FIXME manage port and protocole event if it's an IP address
-        # URL is an IP so save as an IP endpoint
-        elif self.is_valid_ipv4_address(url):
-            try:
-                dupe_endpoint = Endpoint.objects.get(protocol=None,
-                                                     host=url,
-                                                     path=None,
-                                                     query=None,
-                                                     fragment=None,
-                                                     product=finding.test.engagement.product)
-            except:
-                dupe_endpoint = None
-
-            if not dupe_endpoint:
-                endpoints = [Endpoint(host=url, product=finding.test.engagement.product)]
-            else:
-                endpoints = [dupe_endpoint]
-
-            finding.unsaved_endpoints = endpoints
+        endpoint = Endpoint(host=url)
```

Comment:
Could you switch this to `Endpoint.from_uri()` ?

Reply:
Openvas store IP in this field (there is not real URL). Check [mapping](https://github.com/DefectDojo/django-DefectDojo/pull/4188/files/ad2e920f24609ef3686e51aa61adc3769f29667d#diff-d73d11ae11c335f0d99b97145e39ba647b7d856628b0e483cf799ed2e60c9bddR74) and [example scan](https://github.com/DefectDojo/django-DefectDojo/blob/ad2e920f24609ef3686e51aa61adc3769f29667d/dojo/unittests/scans/openvas/report-e2759495-f26d-4089-9c56-12a10dc36c9c.csv#L2)
We can change name of variable from `url` to `ip` or `host`.